src/coreclr/scripts/superpmi.py

```diff
@@ -1618,8 +1618,10 @@ def report_replay_asserts(asserts, output_mch_file):
 
     if asserts:
         logging.info("============================== Assertions:")
+        assertion_num = 0
         for assertion_key, assertion_value in asserts.items():
-            logging.info("%s", assertion_key)
+            assertion_num += 1
+            logging.info("=== Assertion #%s: %s", assertion_num, assertion_key)
             # Sort the values by increasing il size
             sorted_instances = sorted(assertion_value, key=lambda d: d['il'])
             for instance in sorted_instances:
```

Comment:
Can we also limit the amount of examples printed here to the smallest N number of assertions? I sometimes end up with a console full of assertions that I have to scroll up through.